projects/plugins/jetpack/tests/php/bootstrap.php

```diff
@@ -101,6 +101,49 @@ function _manually_load_plugin() {
 }
 
 function _manually_install_woocommerce() {
+	$woo_tests_dir = JETPACK_WOOCOMMERCE_INSTALL_DIR . '/tests';
+	// This is taken from WooCommerce's bootstrap.php file
```

Comment:
This is a lot of stuff to be requiring. Any time one of these files gets removed, or another has to be added, we'd have to update this list. Is there a better way?

Reply:
I'm trying to find one 😓 

Currently I'm experimenting with loading WooCommerce's `bootstrap.php` but there are various errors which I'm not sure how we should handle.

Reply:
I ended up keeping the existing approach (which I agree is not ideal to be clear) plus a few hacks to allow those tests to successfully run 😒